### PR TITLE
Updated traffic lights width and various fixes around it

### LINKED ISSUE #317: [Dashboard] remove 80% width on progress bar. [1]
The dashboard progress bar is set to 80% not sure why.

---

Copied from Trello card: https://trello.com/c/XsRgJUrR/246-dashboard-remove-80-width-on-progress-bar
Labels:

### PR DESCRIPTION
**Pull request by bartaz**
Fix #317 #499 #500

Made traffic lights take whole width, fixed issues with polling and connectors widths.

### CODE REVIEW

File: src/common/components/first-time-heading/index.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ FirstTimeHeading.propTypes = {
 
 function mapStateToProps(state) {
   return {
-    snapsLoaded: state.snaps.success,
+    snapsLoaded: !!state.snaps,
```
Comment (bartaz) [inline src/common/components/first-time-heading/index.js]: Don't rely on `success` flag, as it makes first time progress blink on polling. Fixes #499

File: src/common/components/traffic-lights/traffic-light.css [hunk 2]
```diff
@@ -19,23 +18,24 @@ $signal-colour-default: $light-grey;
...
-  width: 850%;
+  width: 320px;
   background: $signal-colour-default;
   position: absolute;
-  top:50%;
-  left:106%;
-  z-index:-1;
-  margin-top:-2px;
+  top: 50%;
+  left: 30px;
```
Comment (bartaz) [inline src/common/components/traffic-lights/traffic-light.css]: Changed % values into pixels based on signal dimensions to make sure they don't move when width changes.

File: src/common/components/traffic-lights/traffic-light.css [hunk 3]
```diff
@@ -55,7 +55,7 @@ $signal-colour-default: $light-grey;
 }
 
 .box:last-child .signal::after {
-  display:none;
+  background: #FFF; /* cover any line overflowing from previous signals */
```
Comment (bartaz) [inline src/common/components/traffic-lights/traffic-light.css]: Depends on the fact that we have white background, but at least it covers connector overflowing on smaller widths.

File: src/common/components/traffic-lights/traffic-light.css (hunk 2 above)
Comment (earnubs) [inline src/common/components/traffic-lights/traffic-light.css]: Yeah, this was a hurried attempt at making things vaguely responsive that failed.

**Pull request merged**